R/metadata.R

```diff
@@ -5,7 +5,7 @@
 #' via the 'add' command ,or removed with 'remove', and then queried to find
 #' matching objects.
 #'
-#' @param x object, collection or user
+#' @param lpath object, collection or user
```

Comment:
What does iRODS normally use? logical_path? Then I will match that.

Reply:
since we have to continually educate users about logical path vs physical path and that distinction, then yes, I think it's a little cleaner to use all the letters - they can learn from the usage/docs as they read.